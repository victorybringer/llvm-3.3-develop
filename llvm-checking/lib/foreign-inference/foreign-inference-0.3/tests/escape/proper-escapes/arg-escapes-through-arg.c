void f(int * p, int **pp) {
  *pp = p;
}
