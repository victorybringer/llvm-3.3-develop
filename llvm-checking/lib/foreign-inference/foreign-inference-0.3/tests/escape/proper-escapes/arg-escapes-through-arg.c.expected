fromList [ ("@f", fromList [(ArgumentEscape 1, "%p")]) ]
